### PR TITLE
docs: updated gitlab and bitbucket pages with correct links and language. 

### PR DESCRIPTION
**Pull request by robbyandrews**

### CODE REVIEW

File: assets/scss/layouts/_header.scss [hunk 1]
```diff
@@ -168,8 +168,7 @@ button#doks-versions {
 
   .doks-search {
     max-width: 20rem;
-    margin-top: 0.125rem;
-    margin-bottom: 0.125rem;
+    margin: 0 auto;
```
Comment (misterbisson) [inline assets/scss/layouts/_header.scss:171]: This change is unrelated to the PR title and blocked by a question in https://github.com/BluBracket/docs.blubracket.com/pull/97#discussion_r821314162

Comment (robbyandrews) [reply assets/scss/layouts/_header.scss]: I dropped this commit from this PR as well.

**Review by misterbisson — APPROVED** [review]
